MapboxNavigation/Style.swift

```diff
@@ -209,6 +210,115 @@ public class ResumeButton: UIControl {
     }
 }
 
+/// :nodoc:
+@objc(MBSpeedLimitSign)
```

Comment:
This class is big enough (and will only get bigger with full MUTCD and Vienna convention conformance) to justify a separate file.